### PR TITLE
Brazilian portuguese translation

### PR DESCRIPTION
**Pull request by agendator**
Done from scratch, with careful choice of words to better translate into the brazilian context. There are several differences between pt_PT and pt_BR, which in my opinion makes an unified translation quite a mess for both languages. I've also tested it with yay in order to find eventual errors or lack of cohesion for certain terms.

### CODE REVIEW

**Comment by Jguer** [comment]
@guiananias , @agendator has gone through the trouble of doing the pt_BR translation, if it's ok for you will merge this one

**Comment by guiananias** [comment]
@Jguer okay, no problem.

Thanks, @agendator! o/

File: po/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+msgid "error refreshing databases"
+msgstr "error ao recarregar base de dados"
+
+#: clean.go:217 download.go:130
+msgid "error resetting %s: %s"
+msgstr "error ao resetar %s: %s"
+
+#: main.go:111
+msgid "failed to create BuildDir directory '%s': %s"
+msgstr "failed to create BuildDir directory '%s': %s"
```
Comment (guiananias) [inline po/pt_BR.po:519]: @agendator I think you forgot to translate this message here.

Comment (agendator) [reply po/pt_BR.po]: Thanks for spotting it!

**Comment by Jguer** [comment]
> @Jguer okay, no problem. Thanks, @agendator! o/
> 
> Now, a question, should I close that other PR then?

I'll treat the other MR and merge it don't worry.

Thanks for your effort everyone, I'll patch up the conflicts and merge it

**Pull request merged**